src/insights_client/tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+import os
+import pathlib
+import sys
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import _pytest.config
+
+
+def pytest_configure(config: "_pytest.config.Config"):
```

Comment:
while I'm all for type hints, I'm wondering here if referring to the internal pytest type won't possibly break any time in the future; considering `config` is not even used at the moment, IMHO it could be fine to leave it untyped